master/internal/api_tensorboard.go

```diff
@@ -477,10 +477,14 @@ func (a *apiServer) getTensorBoardConfigsFromReq(
 			return nil, err
 		}
 
-		conf, err := db.LegacyExperimentConfigByID(int(expID))
+		conf, unmanaged, err := db.LegacyExperimentConfigForTensorBoard(int(expID))
 		if err != nil {
 			return nil, err
 		}
+		if unmanaged {
+			// cannot create TensorBoard for external/unmanaged experiment
```

Comment:
I'm not sure how this is used on the web

but from like a CLI perspective I think returning an error telling you that you tried to launch against an unmanaged might be better ux

Reply:
Because this is used on a batch of experiments which could be selected over multiple pages,
we'd want to avoid failing the entire op. when some experiments didn't go through.

We probably can even safely ignore an unmanaged experiment being sent to TensorBoard already, so I am not sure about whether this PR needs to go forward